include/simdjson/dom/document_stream.h

```diff
@@ -15,6 +15,7 @@ namespace dom {
 
 
 #ifdef SIMDJSON_THREADS_ENABLED
+/** @private **/
```

Comment:
It is unrelated. I marked it as private because I want to make sure that the worker does not end up in our documentation.

Reply:
That's a good idea. I've been solving that problem either with private nested classes, or by putting these things in the simdjson::internal namespace, which is blocked off from doxygen.